### PR TITLE
Update language names and descriptions

### PR DESCRIPTION
**Pull request by cartermp**
The C#/VB/F# descriptions weren't quite in line with how we're looking to position those languages strategically.  This PR updates that.

### CODE REVIEW

**Comment by ghogen** [comment]
Thanks for making these changes! I added some comments on the text -- if you can make those changes I will merge this PR in.

File: docs/index.md [hunk 1]
```diff
@@ -623,8 +623,8 @@ ms.assetid:
                                                     </div>
                                                 </div>
                                                 <div class="cardText">
-                                                    <h3>Visual C#</h3>
-                                                    <p>A simple, modern, type safe, object oriented programming language used for building applications that run on the .NET Framework.</p>
+                                                    <h3>C#</h3>
+						    <p>A modern Object-Oriented programming language with Functional Programming capabilities for building any application on the .NET platform.</p>
```
Comment (ghogen) [inline docs/index.md]: We would use lowercase for object-oriented and functional programming.

File: docs/index.md [hunk 2]
```diff
@@ -643,7 +643,7 @@ ms.assetid:
                                                 </div>
                                                 <div class="cardText">
                                                     <h3>Visual Basic</h3>
-                                                    <p>A fast, easy to learn programming language you can use to create applications.</p>
+                                                    <p>A modern, easy to learn, Object-Oriented programming language for the .NET platform, focused on easily creating Windows applications.</p>
```
Comment (ghogen) [inline docs/index.md]: and here

File: docs/index.md [hunk 3]
```diff
@@ -680,8 +680,8 @@ ms.assetid:
                                                     </div>
                                                 </div>
                                                 <div class="cardText">
-                                                    <h3>Visual F#</h3>
-                                                    <p>A strongly typed, cross platform programming language that is most often used as a cross platform CLI language, but can also be used to generate JavaScript and GPU code.</p>
+                                                    <h3>F#</h3>
+						    <p>A modern Functional Programming language with Object-Oriented capabilities for the .NET platform, focused on making Functional Programming easier for any task.</p>
```
Comment (ghogen) [inline docs/index.md]: and here

**Comment by cartermp** [comment]
@ghogen addressed.  Thanks for the feedback!

**Pull request merged**